src/ontology/fbbt-edit.obo

```diff
@@ -146642,6 +146644,152 @@ relationship: develops_from FBbt:00059177 ! apolar primary spermatocyte cyst
 property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
 property_value: http://purl.org/dc/terms/date 2022-09-28T18:17:55Z xsd:dateTime
 
+[Term]
+id: FBbt:00059183
+name: somatic cell of larval testis
+def: "A somatic cell that is part of the larval testis. This includes functional cyst cells and precursors of the adult testis epithelium (Mahadevaraju et al., 2021)." [FlyBase:FBrf0248088]
+subset: cur
+is_a: FBbt:00100318 ! somatic cell
```

Comment:
assuming this includes any somatic cells of the larval testis, this could have the logical def - 'somatic cell' that 'part of' some 'larval testis'